### ISSUE TITLE
Script hangs on `Assertion Failure: <unknown>:0:`

### ISSUE THREAD

**Issue by Amzd** [TitouanVanBelle/XCTestHTMLReport#136]
Hangs on some results generated by fastlane and I can't figure out why. It's not the empty logs that was fixed a while ago.

The verbose log is not very detailed:

```
XCTestHTMLReport 1.6.1

Parsing Test Summaries
  ▸ Searching for action_TestSummaries.plist in multi_test_output/iPhoneX/UITests.test_result

Parsing summary
  ▸ Found summary at test_result/1_Test/action_TestSummaries.plist
  ▸ Parsing RunDestination
  ▸ Parsing TargetDevice
  ▸ Parsing TestSummary
(never gets past this)
```
I don't mind it not being able to parse the file for some reason but right now it hangs and blocks the rest of the test flow.

One of the logs that crashes:
[test_result.zip](https://github.com/TitouanVanBelle/XCTestHTMLReport/files/3310810/UITests.test_result.zip)

**Comment by Amzd** [comment]
Ok so I figured out when it hangs. Some tests throw an assertion where it can't figure out what case the assertion came from (I think?) and that looks like this: `Assertion Failure: <unknown>:0:`

It happens on older devices for me but not on iOS 12.

**Comment by Amzd** [comment]
I know this might be an issue with my tests but XCTestHTMLReport should still not hang.

**Comment by TitouanVanBelle** [comment]
This issue has been fixed in 1.7.0. I've tried it on your test results and it seems to work. Closing
